### DISCUSSION TITLE
Sign-up and Login Buttons Don't Do Anything?

### DISCUSSION THREAD

**Discussion by JPDVM2014** [Dan6erbond/share-me#44] [Q&A]
I've got this running, and I went through the setup at my.url/_/, but when trying to sign up or login, the buttons don't function. There is nothing in either log other than listening on port 3000 and the server port info. It looks like the front-end is making requests to the backend, but nothing happens.

Any ideas?

**Comment by rnoxy** [comment]
Look at this https://github.com/Dan6erbond/share-me/issues/42
Adding locations helped a lot.
